Fix - Update SDK docs (#1047)
iotaledger/iota-wiki: next/docs/iota-sdk/docs/welcome.md

![IOTA SDK Overview](/img/banner/client_lib_overview.png)

The IOTA SDK provides a convenient and efficient way to interact with nodes in the Shimmer
and IOTA networks running the [Stardust protocol](https://wiki.iota.org/shimmer/develop/explanations/what-is-stardust).
It consists of two main modules: `client` and `wallet`.

> The `wallet` module uses high-level functions that simplify everyday operations with a standardized interface for

Author: How would you improve it?

> The `wallet` module uses high-level functions that simplify everyday operations with a standardized interface for

Reviewer: I would remove it tbh. It's not useful to explain the inner workings of the code.

> The `wallet` module uses high-level functions that simplify everyday operations with a standardized interface for

Author: @Alex6323 requested it in [this issue](https://github.com/iotaledger/iota-sdk/issues/390). Personally, I think it makes sense to explain why exactly we have the two modules, and what each does. Then again, that could be achieved without the function examples.

module maintains a state. It can optionally interact with [IOTA Stronghold](https://github.com/iotaledger/stronghold.rs/)
for seed handling, storage, and state backup.

The `client` module provides more flexibility and access to low-level functions, which are usually kept "under the hood" by the
`wallet` module. It aims to allow you to customize how you interact with the network in depth, at the cost of
additional complexity.

You can use this documentation to get your application up and running in using the
[programming language of your choice](#available-programming-languages),
[connect to the testnet](explanations/testnet-and-test-tokens.md#connect-to-the-testnet-api),
[explore the network](explanations/testnet-and-test-tokens.md#explore-the-network),
and [get test tokens](explanations/testnet-and-test-tokens.md#get-test-tokens) to develop your application.

## Available Programming Languages

The IOTA SDK is written in [Rust](getting-started/rust.mdx) and also has convenient bindings
in [Node.js](getting-started/nodejs.mdx), [Python](getting-started/python.mdx) and [Wasm](getting-started/wasm.mdx).

> Each of these languages has specific set up instructions you will need to follow to use IOTA SDK in your project. Every
> binding is adjusted for the language's conventions and best practices. For example, Python developers avoid the Builder
> programming pattern, so our Python binding uses named constructor arguments.

Reviewer: ```suggestion
Each of these languages has specific set-up instructions you will need to follow to use the IOTA SDK in your project. The bindings libraries are individually adjusted for the language's conventions and best practices. For example, Python developers avoid the Builder
programming pattern, so the Python binding uses named constructor arguments.
```

## Join the Discussion

> If you want to get involved in discussions about this library, or you're looking for support, go to the `#iota-sdk`
> channel

Reviewer 2: ```suggestion
If you want to get involved in discussions about this library, or you're looking for support, go to the `#libraries`
channel
```
